### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by jtmaveryk** [Labs64/NetLicensing-Community#120] [Questions] [answered]
**Question**
Ask question about NetLicensing services and functionality here.

I was receiving errors signing up with GitHub so I signed up with my email, where would I be able to connect my GitHub and receive the developer pack student perks?

**Additional context**
Add any other context or screenshots related to this question.

**Comment by r-brown** [comment, marked as answer]
Hello @jtmaveryk - please see  _Sunset of GitHub Student Developer Pack and Teacher Toolbox #50_ announcement.
Please send us the needed data as stated in the "How To Apply" section and we'll enrol your vendor account after the verification.

**Comment by r-brown** [comment]
Regarding the GitHub login - your email address needs to be set as public, so NetLicensing authentication system will be able to access and use this; please also refer to  _Github 500 error auth #48_
